static/js/components/EducationTab.js

```diff
@@ -20,7 +20,7 @@ class EducationTab extends React.Component {
     dispatch: Function
   }
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
What was the purpose of this change?

Reply:
componentWillMount is getting deprecated in v16.3.

Reply:
Ah, good to know